mkosi/__init__.py

```diff
@@ -4368,24 +4368,37 @@ def generate_key_cert_pair(args: Args) -> None:
 
 
 def bump_image_version() -> None:
-    """Write current image version plus one to mkosi.version"""
-    version = Path("mkosi.version").read_text().strip()
-    v = version.split(".")
+    """
+    Read current version from 'mkosi.version' and run 'mkosi.bump'
+    or increase version plus one and write it back to 'mkosi.version'
+    """
 
-    try:
-        m = int(v[-1])
-    except ValueError:
-        new_version = version + ".2"
-        logging.info(
-            "Last component of current version is not a decimal integer, "
-            f"appending '.2', bumping '{version}' → '{new_version}'."
-        )
+    if not (version_file := Path("mkosi.version")).exists():
```

Comment:
No need for this to be defined inline, when it's used further below
```suggestion
    version_file = Path("mkosi.version")
    if not version_file.exists():
```